lib/services/crawler/crawler.go

```diff
@@ -41,7 +41,7 @@ var selectors = map[string][]string{
 const urlPattern = "https://www.google.com/search?q=%s"
 
 func (c *Crawler) Run() error {
-	if c.Keyword == nil {
+	if c.Keyword == nil || c.Keyword.Base.ID.ID() == 0 {
```

Comment:
```suggestion
	if c.Keyword == nil || c.Keyword.Base.ID == uuid.NullUUID {
```

Or something similar. Let's not convert the UUID to integers in order to assert if it is null :)

Reply:
@malparty it doesn't work 🙏 
![CleanShot 2023-08-31 at 13 20 28](https://github.com/markgravity/golang-ic/assets/17875522/79889842-c909-4ba1-9904-187ef750ede6)

Reply:
Maybe this 
```suggestion
	if c.Keyword == nil || c.Keyword.Base.ID.ID() == uuid.Nil {
```

Reply:
@suho @malparty it's perfect, I updated it here [c59348a](https://github.com/markgravity/golang-ic/pull/47/commits/c59348a3a013a45a383d8264d0708be0603d55fb)